### ISSUE TITLE
Accessing current location from the marker

### ISSUE THREAD

**Issue by ridyio** [tlserver/flutter_map_location_marker#34]
Hello,
This feature can be really helpful because apparently, if two widgets at the same time request for current location using geolocator, only one of them receives it. It is random. Sometimes the marker can show the current location and geolocator.getCurrentLocation resolves to null or vice versa. This is causing a big unresolvable issue in our app. Hope you can consider some accessor to the current location value that this widget resolves.

**Comment by tlserver** [comment]
Agree, in flutter, it is wired if you are getting some value from some child widget.
